ledger-service/http-json-cli/src/main/scala/com/daml/http/Config.scala

```diff
@@ -160,23 +164,32 @@ private[http] object JdbcConfig
       url <- requiredField(x)("url")
       user <- requiredField(x)("user")
       password <- requiredField(x)("password")
-      createSchema <- optionalBooleanField(x)("createSchema")
+      createSchema <- optionalBooleanField(x)("createSchema").map(
+        _.map { createSchema =>
+          import DbStartupMode._
+          logger.warn(
+            s"The option 'createSchema' is deprecated. Please use for 'createSchema=true' => 'start-mode=${getConfigValue(CreateOnly)}' and for 'createSchema=false' => 'start-mode=${getConfigValue(StartOnly)}'"
```

Comment:
```suggestion
            s"The option 'createSchema' is deprecated. Please use 'start-mode=${getConfigValue(CreateOnly)}' for 'createSchema=true' and 'start-mode=${getConfigValue(StartOnly)}'  for 'createSchema=false'"
```
sounds wrong otherwise but maybe my English is just borked

Reply:
I think you're right, thanks!